homeassistant/components/transmission/__init__.py

```diff
@@ -57,6 +62,12 @@
     vol.Schema(
         {
             vol.Required(CONF_HOST): cv.string,
+            vol.Required(CONF_LIMIT, default=DEFAULT_LIMIT): vol.All(
+                vol.Coerce(int), vol.Range(min=1, max=500)
+            ),
+            vol.Required(CONF_ORDER, default=DEFAULT_ORDER): vol.All(
+                vol.Coerce(str), vol.In(SUPPORTED_ORDER_MODES.keys())
+            ),
```

Comment:
Is it OK to set these to "optional" and then still have the default? I know very little about the config flow stuff but I wonder if this would resolve the CI error?

Reply:
All option keys should be "Optional". Required keys should be the ones required to initialize the integration (like `host`).